option/cc932.c

```diff
@@ -3,7 +3,7 @@
 /* CP932 (Japanese Shift-JIS)                                             */
 /*------------------------------------------------------------------------*/
 
-#include "../ff.h"

```

Comment:
This probably never worked before then, since the header is actualy in `../include/ff.h` although I see that `include/` is being added to the global include path list, so your fix seems correct.

Reply:
Yeah, adding these .c files caused compile errors before changing the `#include` path.